modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/fullscreen_source_editor.js

```diff
@@ -135,9 +135,11 @@ AUI.add(
 				_onEditorChange(event) {
 					var instance = this;
 
-					instance._previewPanel.html(
-						instance._getHtml(event.newVal)
-					);
+					if (event.newVal || event.newVal === '') {
```

Comment:
This seems wrong.

If `event.newVal` is `undefined`, `null`, `0`, they will all pass this conditional.

I think this might be better:

```js
if (event.newVal != null) {
```

Reply:
> If `event.newVal` is `undefined`, `null`, `0`, they will all pass this conditional.

Ignore that, I misread the check as `!== ''`.

Reply:
Hey @wincent,

The intent was to skip the render in case `event.newVal` `undefined`, `null` or `0`, but still render the empty string. I believe this works well now.